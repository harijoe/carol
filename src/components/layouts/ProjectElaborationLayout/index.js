import React from 'react'
import PropTypes from 'prop-types'

import { ProjectElaborationTemplate, HeaderConversational } from 'components'

const ProjectElaborationLayout = ({ children, ...props }) => (
  <ProjectElaborationTemplate header={<HeaderConversational {...props} />}>
    {children}
  </ProjectElaborationTemplate>
)

ProjectElaborationLayout.propTypes = {
  children: PropTypes.any,
}

export default ProjectElaborationLayout
