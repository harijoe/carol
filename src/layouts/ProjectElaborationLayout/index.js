import React from 'react'
import PropTypes from 'prop-types'

import ProjectElaborationTemplate from 'templates/ProjectElaborationTemplate'
import HeaderConversational from './containers/HeaderConversational'

const ProjectElaborationLayout = ({ children, ...props }) =>
  <ProjectElaborationTemplate header={<HeaderConversational {...props} />}>
    {children}
  </ProjectElaborationTemplate>

ProjectElaborationLayout.propTypes = {
  children: PropTypes.any,
}

export default ProjectElaborationLayout
