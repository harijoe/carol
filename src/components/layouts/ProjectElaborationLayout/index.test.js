import React from 'react'
import { shallow } from 'enzyme'

import ProjectElaborationLayout from './'

it('should render ProjectElaborationLayout component', () => {
  expect(shallow(<ProjectElaborationLayout>test children</ProjectElaborationLayout>)).toMatchSnapshot()
})
