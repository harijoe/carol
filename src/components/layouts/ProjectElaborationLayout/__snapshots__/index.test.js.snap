// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ProjectElaborationLayout component 1`] = `
<Mock
  header={<Mock />}
>
  test children
</Mock>
`;
